modules/serverless-functions-key-value-in-configmap-to-env-variable.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-key-value-in-configmap-to-env-variable_{context}"]
+= Setting environment variable from a key value defined in a config map
+
+. Open the `func.yaml` file for your function.
+
+. For each value from a config map key-value pair that you want to assign to an environment variable, add the following YAML to the `envs` section:
```

Comment:
Is _assign_ the right verb (my ignorance)? My reflex thought is to use _create_.

Reply:
You assign a value to a variable. It is true that in the process the variable is created, if it didn't exist before, but there's no need to mention it explicitly.